internal/server/data/migrations.go

```diff
@@ -445,18 +445,18 @@ CREATE UNIQUE INDEX idx_grant_srp ON grants (organization_id,subject,privilege,r
 CREATE UNIQUE INDEX idx_groups_name ON groups (organization_id,name) where (deleted_at is null);
 CREATE UNIQUE INDEX idx_identities_name ON identities (organization_id,name) where (deleted_at is null);
 CREATE UNIQUE INDEX idx_providers_name ON providers (organization_id,name) where (deleted_at is null);
-CREATE UNIQUE INDEX settings_org_id ON settings (organization_id) where deleted_at is null;
+CREATE UNIQUE INDEX IF NOT EXISTS settings_org_id ON settings (organization_id) where deleted_at is null;
```

Comment:
Should the indexes about this also be `IF NOT EXISTS`?

Reply:
not sure what you mean

Reply:
The other `CREATE UNIQUE INDEX` commands *above* this (typo now edited)